### PR TITLE
added functionality for counting tfs 2010 users

### PR DESCRIPTION
**Pull request by aluis94**
Added a new flag -2010 for tfs 2010 user count.
Added a new history function for tfs 2010 that searches through the history of all 18 collections in tfs 2010.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -13,15 +13,27 @@ import (
 
 var collection string
 var minCheckins int
+var is2010 bool
+var tfs2010Collections = [18]string{"CareSystems", "ClaimsEncounters", "CoreSystems","CorpSystemsNon-Erp", "CRM", "DBA","DefaultCollection","Encounters", "E-portal", "ITOps","Octo","ODSDWBI",
```
Comment (raypereda) [inline main.go]: Wrap the long lines. Don't make readers have to scroll right to see what is going on.

**Review by raypereda — CHANGES_REQUESTED** [review]
Run "go fmt" on the code to the format the code. I have that turned on by default in my editor, VS Code.

File: main.go [hunk 2]
```diff
@@ -13,15 +13,27 @@ import (
...
+var rootPath string
 
 func main() {
 	flag.StringVar(&collection, "collection", "http://tfs.molina.mhc:8080/tfs/HSS/", "TFS collection URL")
 	flag.IntVar(&minCheckins, "mincheckins", 10, "minimum number of check-ins for active users")
+	flag.BoolVar(&is2010, "2010", false, "Use the 2010 Repo")
 	flag.Parse()
-
-	h := history()
+	h := make([]string,0)
```
Comment (raypereda) [inline main.go]: This is more idiomatic if you don't know the size 
```go
var h []string
```
See here https://stackoverflow.com/questions/25543520/declare-slice-or-make-slice

File: main.go [hunk 3]
```diff
@@ -42,10 +54,22 @@ func dateRange() string {
...
 }
-
+func history2010()[]string{
+	combined := make([]string,0)
+	collection2010 :="http://dc01tfspv02:8080/tfs/"
+	for _, c := range tfs2010Collections {
+        collection = collection2010+ c;
+		h:= history()
+		if(len(h) > 2){ //if there is a history within the date range
+			combined=append(combined,h[2:]...)
```
Comment (raypereda) [inline main.go]: Nice use of  `... append(combined, h[2:]...)`

**Review by raypereda — CHANGES_REQUESTED** [review]
Address the changes, especially the `go fmt`, and I'll review again.

File: main.go [hunk 4]
```diff
@@ -14,24 +14,25 @@ import (
 var collection string
 var minCheckins int
 var is2010 bool
-var tfs2010Collections = [18]string{"CareSystems", "ClaimsEncounters", "CoreSystems","CorpSystemsNon-Erp", "CRM", "DBA","DefaultCollection","Encounters", "E-portal", "ITOps","Octo","ODSDWBI",
-"PRELEX","Provider-Eligibility-Extracts","QNXT-Reports-Utilities","Security","SOA","TCIM"}
+var tfs2010Collections = [18]string{"CareSystems", "ClaimsEncounters", "CoreSystems", "CorpSystemsNon-Erp", "CRM",
```
Comment (aluis94) [inline main.go:17]: Word wrapped this string and ran go fmt

File: main.go [hunk 5]
```diff
@@ -14,24 +14,25 @@ import (
...
 var rootPath string
 
 func main() {
 	flag.StringVar(&collection, "collection", "http://tfs.molina.mhc:8080/tfs/HSS/", "TFS collection URL")
 	flag.IntVar(&minCheckins, "mincheckins", 10, "minimum number of check-ins for active users")
 	flag.BoolVar(&is2010, "2010", false, "Use the 2010 Repo")
 	flag.Parse()
-	h := make([]string,0)
-	if(!is2010){
+	var h []string
```
Comment (aluis94) [inline main.go:27]: Changed string array initialization

**Comment by raypereda** [comment]
:+1: Nice job @aluis94

**Pull request merged**